alan_std.js

```diff
@@ -1083,15 +1085,15 @@ export async function runWindow(initialContextFn, contextFn, gpgpuShaderFn) {
     canvas.style['height'] = '100%';
     canvas.width = window.innerWidth;
     canvas.height = window.innerHeight;
-    document.body.addEventListener("resize", () => {
+    window.addEventListener("resize", () => {
```

Comment:
Apparently this event never fires on the `body`, only the top-level `window` object. (Especially confusing because I do remember seeing documentation stating otherwise.)